lightning/backend/session.go

```diff
@@ -60,9 +60,12 @@ func (t *transaction) Iter(k kv.Key, upperBound kv.Key) (kv.Iterator, error) {
 
 // Set implements the kv.Mutator interface
 func (t *transaction) Set(k kv.Key, v []byte) error {
+	val := make([]byte, len(v))
+	copy(val, v)
+
 	t.kvPairs = append(t.kvPairs, common.KvPair{
 		Key: k.Clone(),
-		Val: append([]byte{}, v...),
+		Val: val,
```

Comment:
remain append: https://github.com/pingcap/tidb-lightning/pull/281/commits/430f66a54c796071bb2491a440e47bd34554c2af
benchmark tells it will not faster..